js/ui/date_range_box.js

```diff
@@ -0,0 +1,33 @@
+import registerComponent from '../core/component_registrator';
+import { extend } from '../core/utils/extend';
+import DateBox from './date_box';
+
+const DateRangeBox = DateBox.inherit({
```

Comment:
Seems comment
// STYLE dateRangeBox 
is missed. I think it will be useful in the future. All public components has this comment.

Reply:
hm, missed this, ty